eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -63,7 +63,7 @@ object Channel {
   val MAX_NEGOTIATION_ITERATIONS = 20
 
   // this is defined in BOLT 11
-  val MIN_CLTV_EXPIRY_DELTA = CltvExpiryDelta(9)
+  val MIN_CLTV_EXPIRY_DELTA = CltvExpiryDelta(18)
```

Comment:
the comment above is inaccurate now.

Reply:
Depends, I'm hoping we merge https://github.com/lightningnetwork/lightning-rfc/pull/785 soon(ish) so this value will be in Bolt 11 :D